editoast/fga_migrations/src/main.rs

```diff
@@ -0,0 +1,93 @@
+pub mod cli;
+
+use crate::cli::ApplyArgs;
+use crate::cli::CliParser;
+use crate::cli::Commands;
+use common::tracing::Stream;
+use common::tracing::Telemetry;
+use common::tracing::TracingConfig;
+use common::tracing::create_tracing_subscriber;
+use fga::Client;
+use fga::client::ConnectionSettings;
+use fga_migrations::MigrationError;
+use fga_migrations::TargetMigration;
+use fga_migrations::run_migrations;
+
+use opentelemetry_otlp::WithExportConfig;
+use tracing_subscriber::util::SubscriberInitExt;
+
+use clap::Parser;
+use url::Url;
+
+fn setup_telemetry(telemetry_url: Url) {
+    let exporter = opentelemetry_otlp::SpanExporter::builder()
+        .with_tonic()
+        .with_endpoint(telemetry_url.as_str())
+        .build()
+        .expect("failed to build a span exporter");
+
+    let telemetry = Some(Telemetry {
+        service_name: "fga-migrations".to_string(),
+        endpoint: telemetry_url,
+    });
+
+    let tracing_config = TracingConfig {
+        stream: Stream::Stderr,
+        telemetry,
+        directives: vec![],
+    };
+    create_tracing_subscriber(
+        tracing_config,
+        tracing_subscriber::filter::LevelFilter::INFO,
+        exporter,
+    )
+    .init();
+}
+
+async fn retrieve_openfga_clients(
+    authz_store: &str,
+    migrations_store: &str,
+    connection_settings: &ConnectionSettings,
```

Comment:
```suggestion
    authz_store: String,
    migrations_store: String,
    connection_settings: ConnectionSettings,
```

no point in demanding refs just to clone it

Reply:
Same as the thread above: that part does not exist anymore. Resolving the thread.